config/splat.us.stlib.yaml

```diff
@@ -63,9 +65,10 @@ segments:
       - [0x41898, c, e_red_door]
       - [0x42548, c, e_room_fg]
       - [0x42634, c, st_common]
+      - [0x43AE4, c, e_misc]
       - [0x45E80, c, e_collect]
       - [0x473EC, c, blit_char]
-      - [0x4769C, c, e_misc]
+      - [0x4769C, c, e_misc_2]
```

Comment:
This is more evidence to me that `BlitChar`, `EntityRelicOrb`, `EntityHeartDrop`, `EntityMessageBox` likely reside within `e_collect`.